lib/serialport.js

```diff
@@ -493,4 +478,10 @@ SerialPort.prototype.drain = function(callback) {
   }.bind(this));
 };
 
-module.exports = factory;
+SerialPort.parsers = parsers;
+SerialPort.list = SerialPortBinding.list;
+
+// TODO deprecate
+SerialPort.SerialPort = SerialPort;
```

Comment:
Something like? 
```js
Object.defineProperty(SerialPort, 'SerialPort', {
  get: function() { 
    console.warn('DEPRECATION: Please use `require(\'serialport\')` instead of `require(\'serialport\').SerialPort`');
    Object.defineProperty(SerialPort, 'SerialPort', { value: SerialPort });
    return SerialPort;
  },
  configurable: true
});
```